Cargo.toml

```diff
@@ -15,7 +15,7 @@ features = ["bevy_winit/x11", "immutable_ctx"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-default = ["manage_clipboard", "open_url", "default_fonts", "render", "picking"]
```

Comment:
One minor gotcha here is that `bevy_ui` depends on `render`.. We might want to either split it into a separate feature or move to using explicit optional-dependency features (`dep:`).

Then again, this is arguably pretty benign.. if you're not rendering then there is no order conflict anyway.